cli/Dockerfile-bin

```
@@ -1,5 +1,9 @@
 ## compile binaries
-FROM gcr.io/linkerd-io/go-deps:ceb2d9f1 as golang
+FROM gcr.io/linkerd-io/go-deps:da6a119d as golang
+
+# Install packr in order to use custom build command for static files
+RUN CGO_ENABLED=0 go get -v github.com/gobuffalo/packr/v2/packr2
```

Related to [@siggy's comment](https://github.com/linkerd/linkerd2/pull/2172#discussion_r252066858), this is pulling the source from github, rather than installing it from the filesystem. That means we could end up with a version of packr that's not pinned to the version specified in Gopkg.toml, which isn't ideal. This should fix it:

```suggestion
RUN go install ./go/src/github.com/gobuffalo/packr/v2/packr2
```

---

I see - thanks!